src/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -444,7 +444,7 @@ def update_progress(cur_loss, iteration):
 
         }
         model = _tc.extensions.object_detector()
-        model.train(data=dataset, annotations_column_name=annotations, image_column_name=feature, options=tf_config)
+        model.train(data=dataset, annotations_column_name=annotations, image_column_name=feature, class_labels=classes, options=tf_config)
```

Comment:
Please add `class_labels` to options. I doesn't feel like a top-level argument to me